### DISCUSSION TITLE
在线等

### DISCUSSION THREAD

**Discussion by nahida0-o** [share121/inter-knot#1807] [其他]
microsoft edge这个浏览器怎么举报某网页？刚才用这个浏览器再次登陆绳网的时候想着直接恢复之前的页面就行，结果不知道为什么弹出来了个黄色网站。找了一圈没找到举报键在哪，还是说压根没有举报功能？这个黄色网站还是要会员的那种，一看就是来诈骗的。在线等，很急。

**Comment by EMOJT64** [comment]
EDGE好像是没有举报功能的。。。

**Reply by nahida0-o** [reply]
感谢解答，那只能顺从了

**Reply by nahida0-o** [reply]
欸？不对啊，我记得之前在一次更新绳网的时候弹出来个界面告诉我当前网站被举报了（虽然好像还是能进？）。算了，我对电脑基本一窍不通就不捣鼓了。（口水派蒙）

**Comment by sky-shunfengjun** [comment]
可以点击右上角三个点-帮助和反馈-举报不安全的站点

**Comment by Dave-12138** [comment]
猜你想问：
![image](https://github.com/user-attachments/assets/ca6dde5c-886c-4364-80f1-0fe408934d9b)
笑点解析：其实是报告为不安全（举报钓鱼网站或者报告有病毒），幽默微软中文。
当然，你知道的，浏览器只是浏览器，微软没有对互联网的审查与管理权，不可能因为举报把网页干掉，这个功能的作用就是让别人用edge进被举报网页的时候被拦一下。

**Comment by yanlei2009** [comment]
浏览器本身好像不带举报功能
